BlueprintUI/Sources/Element/MeasurableStorage.swift

```diff
@@ -17,7 +28,19 @@ extension MeasurableStorage: CaffeinatedContentStorage {
         environment: Environment,
         node: LayoutTreeNode
     ) -> CGSize {
-        measurer(proposal, environment)
+        guard environment.layoutMode.options.measureableStorageCache, let validationKey else {
+            return measurer(proposal, environment)
+        }
+
+        let key = MeasurableSizeKey(path: node.path, max: proposal.maximum)
+        return environment.cacheStorage.measurableStorageCache.retrieveOrCreate(
```

Comment:
Thought: Could we stick making the key and looking up the value on `LayoutTreeNode` (in a lazy way) so that we don't need to do this key creation and lookup dance outside of a single spot / in every `Storage` that will need it?